storage/src/robocorp/storage/__init__.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import random
+import time
+from typing import Dict, List, Optional
+
+from ._requests import Requests, RequestsHTTPError
+from ._storage import AssetNotFound
+from ._storage import get_assets_client as _get_assets_client
+from ._utils import with_lazy_objects as _with_lazy_objects
+
+__version__ = "0.1.0"
+version_info = [int(x) for x in __version__.split(".")]
+
+LOGGER = logging.getLogger(__name__)
+_with_lazy_assets = _with_lazy_objects(_assets_client=_get_assets_client)
```

Comment:
I get it, but it's a bit of a roundabout way of doing this. You could also `@cache` decorate `get_assets_client` and then call that in functions whenever you need the client, instead of injecting it as arguments.

Reply:
Yes, moved to lru-cached function call everytime is needed, best way to do it would be as an adapter property in a class once we translate this into adapter so it can be mirrored by the local files one. Thx!

Reply:
Yes, I think the singleton / cached client should support adapters later on, and that can be controlled by an env var.

As a sidenote, if we need a way in-code to set the adapter, we can copy the configure pattern from the browser library. So something like:
```python
from robocorp import storage

def some_task():
    storage.configure(adapter=storage.FileAdapter)
    storage.set_asset(...)
```